submissions/bettyBada/html-forms/signup.js

```diff
@@ -0,0 +1,97 @@
+let submitForm = document.getElementById("validateForm");
+let firstName = document.getElementById("firstName");
+let lastName = document.getElementById("lastName");
+let otherName = document.getElementById("otherName");
+let phoneNumber = document.getElementById("number");
+let email = document.getElementById('email');
+let password = document.getElementById("password");
+let confirmPassword = document.getElementById("confirmPassword");
+
+function firstNameValidation(field, event) {
+    let regExp = /\W/;
```

Comment:
Does this regex expression cover all unwanted characters a user might use?

Reply:
Yes, it covers it.
<img width="837" alt="Screenshot 2021-04-22 at 17 05 12" src="https://user-images.githubusercontent.com/50108236/115747197-e8b72e00-a38c-11eb-9acf-1bbca17b0146.png">